lib/gateway/services/loki.conf.tpl

```diff
@@ -0,0 +1,16 @@
+location /api/loki { return 302 https://${BENTOV2_PORTAL_DOMAIN}/api/loki/; }
+location /api/loki/ {
+    # Reverse proxy settings
+    include /gateway/conf/proxy.conf;
+    include /gateway/conf/proxy_extra.conf;
+    include /gateway/conf/proxy_private.conf;
+
+    # Forward request to the aggregation
+    rewrite ^ $request_uri;
+    rewrite ^/api/loki/(.*) /$1 break;
+    return 400;
+    proxy_pass http://bentov2-loki:3100/loki/api/v1$uri;
```

Comment:
use the `BENTO_LOKI_CONTAINER_NAME` environment variable instead of hard-coding the container name